src/Components/ProjectUnderdog.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import globalStyles from '../CSS/globalStyles.module.css';
+import styles from '../CSS/ProjectUnderdog.module.css';
+import { Row, Col } from 'antd';
+import underdog_hero from '../Assets/underdog_hero.jpeg';
```

Comment:
Is this image free to use? I just want to make sure this image doesn't come with any licensing issues.

Reply:
I asked about that Sunday. He said it is from a free clip art source.